teachertool/src/components/EvalResultDisplay.tsx

```diff
@@ -0,0 +1,26 @@
+/// <reference path="../../../built/pxtblocks.d.ts"/>
+
+import { useContext, useState } from "react";
+import { AppStateContext } from "../state/appStateContext";
+
+interface IProps {}
+
+const EvalResultDisplay: React.FC<IProps> = ({}) => {
+    const { state: teacherTool } = useContext(AppStateContext);
+
+    return (
+        <>
+            {teacherTool.projectMetadata && (
```

Comment:
I'm not a huge fan of having this whole component be conditional. Can we have this check in `App.tsx` instead and just have `EvalResultDisplay` return the div?

Reply:
I wouldn't worry too much about how this component works just yet. This is placeholder while we get a design worked out.